src/main/java/net/mcreator/ui/dialogs/AcceleratorDialog.java

```diff
@@ -47,16 +47,15 @@ public AcceleratorDialog(MCreator mcreator) {
 
 		List<JAcceleratorButton> buttonsList = new ArrayList<>();
 
-		JPanel northPanel = new JPanel(new BorderLayout(20, 20));
-		northPanel.add(L10N.label("dialog.accelerators.description"), "North");
+		List<BasicAction> actions = mcreator.actionRegistry.getActions().stream()
+				.filter(a -> a instanceof BasicAction ba && ba.getAccelerator() != null).map(a -> (BasicAction) a)
+				.toList();
 
-		JComboBox<String> sections = new JComboBox<>(
-				AcceleratorsManager.INSTANCE.SECTIONS.stream().map(s -> L10N.t("dialog.accelerators.section." + s))
-						.toArray(String[]::new));
+		JTabbedPane tabbedPane = new JTabbedPane(JTabbedPane.TOP);

```

Comment:
[This](https://github.com/MCreator/MCreator/blob/5405949d8b22cdcc747dc1117cd07ee5bc96bba6/src/main/java/net/mcreator/ui/modgui/codeviewer/ModElementCodeViewer.java#L48) will look nicer:
```suggestion
		JTabbedPane tabbedPane = new JTabbedPane(JTabbedPane.TOP, JTabbedPane.SCROLL_TAB_LAYOUT);
```

Reply:
I tested it and it's not.